Content.Server/Cargo/Components/CargoPalletComponent.cs

```diff
@@ -4,4 +4,11 @@ namespace Content.Server.Cargo.Components;
 /// Any entities intersecting when a shuttle is recalled will be sold.
 /// </summary>
 [RegisterComponent]
-public sealed partial class CargoPalletComponent : Component {}
+public sealed partial class CargoPalletComponent : Component
+{
+    /// <summary>
+    /// Whether the pad is a buy pad, a sell pad, or both.
+    /// </summary>
+    [DataField("pallettype", required: false)]
+    public string PalletType;
```

Comment:
This should be an Enum instead of a string

Reply:
I can't find a single example of someone using datafield with an enum anywhere, and it's complaining when I try. Are you sure about this

`    [DataField]
    public enum PalletType {buy, sell, both}`
Attribute 'DataField' is not valid on this declaration type. It is only valid on 'property, indexer, field' declarations. [Content.Server]

Reply:
Ah, not quite.

Define the enum type elsewhere (probably the bottom of the file)
```C#
public enum BuySellType
{
   Both,
   Buy,
   Sell
}
```

Then use that as the type for the DataField

```C#
[DataField]
public BuySellType PalletType;
```

If that doesn't work, let me know. You might need to specify a custom serializer for the DataField.